app/renderer/js/components/tab.ts

```diff
@@ -29,7 +31,7 @@ class Tab extends BaseComponent {
 	}
 
 	isLoading(): boolean {
-		return this.webview.isLoading;
+		return this.webview.$el.isLoading();
```

Comment:
Can you explain why this change is correct? Is this a bug from before?

Reply:
@priyank-p Yes... ```isLoading``` is a property of ```<webview>``` type but here ```this.webview``` was of type ```WebView``` class which we created. ```$el``` is of type ```<webview>``` . If this is correct, then yes, this should be a bug from before.. @abhigyank and @akashnimare Can you please confirm this?

Reply:
From documentation looks correct. Does this log correct value when called?

Reply:
When i checked this, i coudn't exactly test out its working, but on console.logging previous code return ```undefined```, while this returns ```boolean``` values.